packages/app/contexts/branch.tsx

```diff
@@ -13,9 +13,11 @@ import branch from 'react-native-branch';
 import { DEFAULT_LURE, DEFAULT_PRIORITY_TOKEN } from '../constants';
 import storage from '../lib/storage';
 import { getPathFromWer } from '../utils/string';
+import { useShip } from './ship';
 
 interface LureData extends DeepLinkMetadata {
   id: string;
+  clickedPreAuth: boolean;
```

Comment:
Either a comment or a different variable name would be helpful here -- not obvious to me what "clickedPreAuth" means.